src/EnergyPlus/PluginManager.cc

```diff
@@ -402,7 +403,7 @@ namespace PluginManagement {
 #endif
     }
 
-    void registerNewCallback(int iCalledFrom, const std::function<void()> &f)
+    void registerNewCallback(EnergyPlusData &EP_UNUSED(state), int iCalledFrom, const std::function<void(void *)> &f)
```

Comment:
Also note here the function form that is passed in as a callback is now: `const std::function<void(void *)> &`, so the callback must be a void function that accepts a void pointer as a single argument.